src/Index.jsx

```diff
@@ -1,25 +1,59 @@
+/* global TERRA_DEV_SITE_BASENAME */
+// TERRA_DEV_SITE_BASENAME is defined by webpack
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { HashRouter as Router } from 'react-router-dom';
+
+import ApplicationBase from 'terra-application';
+import ApplicationNavigation from 'terra-application-navigation';
+
 // This line will be resolved by webpack
 // eslint-disable-next-line import/no-unresolved, import/extensions
-import siteConfig from 'build/appConfig';
+import siteConfig from 'build/siteConfig';
 
-import App from './app/App';
+import Site from './site/_Site';
+
+// eslint-disable-next-line import/no-unresolved, import/extensions
+const fetchSearchItems = () => import(/* webpackPrefetch: true, webpackChunkName: "build/searchItems" */ 'build/searchItems').then(({ default: items }) => items);
```

Comment:
Seems like a candidate for `React.lazy()`. Why did we use webpack's dynamic chunk loading syntax here instead?

Reply:
This guy is just loading a plain javascript object, not a react component so standard syntax applies.